collect_app/src/main/java/org/odk/collect/android/activities/AppListActivity.java

```diff
@@ -189,6 +193,11 @@ private void setupDrawerItems() {
             public View getView(int position, View convertView, ViewGroup parent) {
                 TextView textView = (TextView) super.getView(position, convertView, parent);
                 textView.setPadding(50, 0, 0, 0);
+                if ((position == 0)&&(mFirstTime)) {
```

Comment:
By “add no value”, I was including readability. I see superfluous symbols as noise that slows down reading. But reasonable people may disagree.